app/views/rentals/_search_users_table.html.erb

```diff
@@ -0,0 +1,26 @@
+<%= will_paginate @users, renderer: BootstrapPagination::Rails %>
+<table class="table table-striped table-bordered table-hover">
+  <thead>
+    <tr>
+      <th>First name</th>
+      <th>Last name</th>
+      <th>Email</th>
+      <th>Spire</th>
+      <th>Department</th>
+      <th class="min-width">Actions</th>
```

Comment:
Ref: https://github.com/umts/probable-engine/blob/master/app/assets/stylesheets/tables.css#L21-L24

So I'm still confused though. Why do you want a 1px-wide table column?